supertokens_python/recipe/multitenancy/interfaces.py

```diff
@@ -175,7 +175,9 @@ def __init__(
 
 
 class ThirdPartyProvider:
-    def __init__(self, id: str, name: Optional[str]):
+    def __init__(
+        self, id: str, name: Optional[str]
+    ):  # pylint: disable=redefined-builtin
```

Comment:
what does this comment mean?

Reply:
We are redefining the variable `id` which is also a built in function that gives you the mem location of any object. 

```python
╰─❯ python                                         
Python 3.10.6 (main, May 29 2023, 11:10:38) [GCC 11.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> x = 5
>>> id(x)
140504367890800
```

It won't cause any issue unless you need to use the original id function (which we don't). So pylint wants you to avoid it by default. But to keep it consistent it other SDKs, I disabled pylint here.